.github/workflows/run_detekt_and_unit_tests.yml

```diff
@@ -37,12 +37,12 @@ jobs:
           name: DetektReportsTemplate
           path: template/build/reports/detekt/
 
-      - name: Run unit tests and Jacoco on template
+      - name: Run unit tests with Kover on template
         working-directory: ./template
-        run: ./gradlew jacocoTestReport
+        run: ./gradlew koverMergedXmlReport
```

Comment:
Woops, this should be `koverMergedHtmllReport` too 🙇 

Side question: Will you make the changes for sample in a separate pull request?

Reply:
Sounds good, please don't forget to update the PR description 😁